### DISCUSSION TITLE
How long are artifacts available?

### DISCUSSION THREAD

**Discussion by nilsreichardt** [codemagic-ci-cd/codemagic-docs#1606] [Q&A] [answered]
When I have an artifact of my build and I make this build public with via the [Codemagic API](https://docs.codemagic.io/rest-api/artifacts/#step-2-create-a-public-download-url-using-the-url-obtained-in-step-1), how long is the build available? I can pass a `expiresAt` timestamp, but can it be the case that the artifact is deleted before it expires?

**Comment by dtrdic** [comment, marked as answer]
Hey Nils,
well, the build availability actually depends on your account type. You can check it more on this page
https://docs.codemagic.io/billing/pricing/#build-history-and-artifact-storage

**Reply by nilsreichardt** [reply]
Hey @dtrdic, thanks I was searching for this link! <3
